jac-uk/apply
# jac-uk/admin#1974 JAC00164 Medical Members - Application Changes

## What's included?
Changes for JAC00164 
see https://judicial-appointments.slack.com/archives/C02QHCFD10C/p1682440284122459

Closes jac-uk/admin#1974

## Who should test?
✅ Product owner
✅ Developers
✅ Exercise team

## How to test?
Apply for one of the following vacancies and see that the application form includes the changes in  jac-uk/admin#1974:

- Develop: JAC00507
- Staging: JAC00660
- Production: JAC00164

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:STAGING
_can be OFF, DEVELOP or STAGING_
